test/resolve/resolver_test.go

```diff
@@ -3,70 +3,62 @@ package resolve
 import (
 	"os"
 	"path/filepath"
-	"strings"
 	"testing"
 
 	"github.com/debricked/cli/internal/cmd/resolve"
 	"github.com/debricked/cli/internal/wire"
 	"github.com/stretchr/testify/assert"
 )
 
-func removeLines(input, prefix string) string {
-	lines := strings.Split(input, "\n")
-	var result []string
-
-	for _, line := range lines {
-		if !strings.HasPrefix(line, prefix) {
-			result = append(result, line)
-		}
-	}
-
-	return strings.Join(result, "\n")
-}
-
 func TestResolves(t *testing.T) {
 	cases := []struct {
 		name           string
 		manifestFile   string
 		lockFileName   string
-		expectedFile   string
 		packageManager string
 	}{
 		{
 			name:           "basic package.json",
 			manifestFile:   "testdata/npm/package.json",
 			lockFileName:   "yarn.lock",
-			expectedFile:   "testdata/npm/yarn-expected.lock",
 			packageManager: "npm",
 		},
 		{
 			name:           "basic requirements.txt",
 			manifestFile:   "testdata/pip/requirements.txt",
 			lockFileName:   "requirements.txt.pip.debricked.lock",
-			expectedFile:   "testdata/pip/expected.lock",
 			packageManager: "pip",
 		},
 		{
 			name:           "basic .csproj",
 			manifestFile:   "testdata/nuget/csproj/basic.csproj",
 			lockFileName:   "packages.lock.json",
-			expectedFile:   "testdata/nuget/csproj/packages-expected.lock.json",
 			packageManager: "nuget",
 		},
 		{
 			name:           "basic packages.config",
 			manifestFile:   "testdata/nuget/packagesconfig/packages.config",
 			lockFileName:   "packages.config.nuget.debricked.lock",
-			expectedFile:   "testdata/nuget/packagesconfig/packages.config.expected.lock",
 			packageManager: "nuget",
 		},
 		{
 			name:           "basic go.mod",
 			manifestFile:   "testdata/gomod/go.mod",
 			lockFileName:   "gomod.debricked.lock",
-			expectedFile:   "testdata/gomod/expected.lock",
 			packageManager: "gomod",
 		},
+		{
+			name:           "basic pom.xml",
+			manifestFile:   "testdata/maven/pom.xml",
+			lockFileName:   "maven.debricked.lock",
+			packageManager: "maven",
+		},
+		{
+			name:           "basic build.gradle",
+			manifestFile:   "testdata/gradle/build.gradle",
+			lockFileName:   "gradle.debricked.lock",
+			packageManager: "gradle",
+		},
```

Comment:
Changed this test to not assert exact contents of the resolved file, but rather just existence. lockfiles are different depending on OS, which makes them too dynamic.

Reply:
I think that goes a very long way :+1: